jerk_weight: 10.0         # weight for "smoothness" cost for jerk
over_v_weight: 100000.0   # weight for "over speed limit" cost
over_a_weight: 5000.0     # weight for "over accel limit" cost
over_j_weight: 2000.0     # weight for "over jerk limit" cost
